build_release.py

```
@@ -1,60 +1,147 @@
-for deployment, code in zip(config["dataContracts"], metadata["bytecodes"]):
-    deploymentInfo["dataContracts"].append(
-        {
-            "address": deployment["address"],
-            "salt": deployment["salt"],
-            "nonce": deployment["nonce"],
-            "initcode": "0x" + code,
-        }
-    )
+
+if metadata is not None:
+    for deployment, code in zip(config["dataContracts"], metadata["bytecodes"]):
```

Ruff (0.15.7)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)